### PR TITLE
Edit to be made when setting up commudle Locally

### PR DESCRIPTION
**Pull request by imrrahul**
I have made following edits while setting up Commudle in Local Machine : 

1.) Go to ./config/storage.yml and add the following codes :  

`test:
  service: Disk
  root: <%= Rails.root.join("tmp/storage") %>

local:
  service: Disk
  root: <%= Rails.root.join("storage") %>`

and

2.)  If you are using local machine instead amazone aws just edit the following code (if not already done ) in ./config/environments/development.rb : 
`config.active_storage.service = :local` 

instead of : 
`config.active_storage.service = :amazon`

### CODE REVIEW

**Comment by imrrahul** [comment]
@arpansac review this once!!

**Comment by imrrahul** [comment]
@arpansac  review this once.

**Comment by arpansac** [comment]
@imrrahul I'll just need time till Sunday, I know things are getting delayed, will be back soon! :)

**Comment by arpansac** [comment]
@imrrahul I'll merge this today! Thanks a ton.

**Comment by imrrahul** [comment]
cool thanks!! @arpansac

File: config/storage.yml [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+
+\\ When using AWS :
 amazon:
   service: S3
   access_key_id: <%= ENV['KOMMUNITY_APP_S3_ACCESS_ID'] %>
   secret_access_key: <%= ENV['KOMMUNITY_APP_S3_SECRET'] %>
   region: ap-south-1
-  bucket: <%= ENV['KOMMUNITY_APP_S3_BUCKET'] %>
\ No newline at end of file
+  bucket: <%= ENV['KOMMUNITY_APP_S3_BUCKET'] %>
```
Comment (arpansac) [inline config/storage.yml]: @imrrahul Let's merge this. Also, please do take care of redundant changes!

**Comment by arpansac** [comment]
@imrrahul I've created an issue for this and edited the body of the PR to test auto closing of issues

**Pull request merged**